PolyEngine/RenderingDevice/OpenGL/CMakeLists.txt

```diff
@@ -29,10 +29,24 @@ set(POLYGLDEVICE_H_FOR_IDE
 	Src/Text2DRenderingPass.hpp
 	Src/TransparentRenderingPass.hpp
 )
+set(POLYGLDEVICE_LIBS polycore polyengine epoxy::gl)
+
+# macOS Obj-C sources
+if(MACOSX)
+	set(POLYGLDEVICE_SRCS
+			${POLYGLDEVICE_SRCS}
+            Src-macOS/CocoaImpl.mm)
+	set(POLYGLDEVICE_INCLUDE ${POLYGLDEVICE_INCLUDE} Src-macOS)
```

Comment:
You can use generator expressions to include the macOS source dir inline:
`set(POLYGLDEVICE_INCLUDE Src $<$<BOOL:${APPLE}>:Src-macOS>)`